packages/fxa-auth-server/lib/senders/emails/templates/fraudulentAccountDeletion/en.ftl

```diff
@@ -1,7 +1,10 @@
 fraudulentAccountDeletion-subject = Your { -product-firefox-account } was deleted
+fraudulentAccountDeletion-subject-2 = Your { -product-mozilla-account } was deleted
 fraudulentAccountDeletion-title = Your account was deleted
 fraudulentAccountDeletion-content = Recently, a { -product-firefox-account } was created and a subscription was charged using this email address. As we do with all new accounts, we asked that you confirm your account by first validating this email address.
+fraudulentAccountDeletion-content-moz = Recently, a { -product-mozilla-account } was created and a subscription was charged using this email address. As we do with all new accounts, we asked that you confirm your account by first validating this email address.
 fraudulentAccountDeletion-content-2 = At present, we see that the account was never confirmed. Since this step was not completed, we are not sure if this was an authorized subscription. As a result, the { -product-firefox-account } registered to this email address was deleted and your subscription was canceled with all charges reimbursed.
+fraudulentAccountDeletion-content-2-moz = At present, we see that the account was never confirmed. Since this step was not completed, we are not sure if this was an authorized subscription. As a result, the { -product-mozilla-account } registered to this email address was deleted and your subscription was canceled with all charges reimbursed.
```

Comment:
```suggestion
fraudulentAccountDeletion-content-3 = At present, we see that the account was never confirmed. Since this step was not completed, we are not sure if this was an authorized subscription. As a result, the { -product-mozilla-account } registered to this email address was deleted and your subscription was canceled with all charges reimbursed.
```

Nit: only this and one other string are updated using -moz. Suggestion is for consistency.

Reply:
See comment above  - we need to leave `fraudulentAccountDeletion-content` and `fraudulentAccountDeletion-content-2` untouched, so I've replaced the new string IDs with  `fraudulentAccountDeletion-content-part1-v2` and `fraudulentAccountDeletion-content-part2-v2`

Reply:
Your change makes sense and IDs are easier to understand 👍